components/ui/__tests__/button.test.tsx

```diff
@@ -92,18 +85,9 @@ describe("Button", () => {
 
   describe("SecondaryButton", () => {
     it("renders correctly", () => {
-      const { container } = render(
-        <SecondaryButton>test button</SecondaryButton>,
-      );
+      render(<SecondaryButton>test button</SecondaryButton>);
 
-      expect(container.firstChild).toMatchInlineSnapshot(`
-        <button
-          class="inline-flex flex-row px-4 py-2 text-sm rounded-md items-center justify-center border border-transparent font-medium focus:outline-none focus:ring-2 focus:ring-offset-2 text-blue-700 bg-blue-100 hover:bg-blue-200 focus:ring-blue-500 disabled:cursor-not-allowed disabled:opacity-75"
-          type="button"
-        >
-          test button
-        </button>
-      `);
+      expect(screen.getByText("test button")).toBeInTheDocument();
```

Comment:
I think we can keep the .toMatchInlineSnapshot() usage here since we're actually testing the styling for this component and the snapshot result is very small and readable.